drizzlepac/pars/hap_pars/svm_parameters/acs/hrc/acs_hrc_astrodrizzle_any_n6.json

```diff
@@ -22,7 +22,7 @@
         "skystat": "median",
         "skywidth": 0.1,
         "skylower": -100.0,
-        "sky_bits": "0",
+        "sky_bits": "64646464
```

Comment:
Looks like a double quote is needed here.

Reply:
oh thanks for catching this. "sky_bits" actually needs to be just "64" not "6464646464646....."